diesel/src/migrations/migration.rs

```diff
@@ -101,6 +109,7 @@ fn run_sql_from_file(conn: &SimpleConnection, path: &Path) -> Result<(), RunMigr
 }
 
 #[cfg(test)]
+#[cfg_attr(feature = "clippy", allow(result_unwrap_used))]
```

Comment:
Isn't this redundant with https://github.com/diesel-rs/diesel/pull/305/files#diff-7ab3c70a207718b2fce878a8f925a415R22?